### PR TITLE
Version tags, Fedora 15

### PR DESCRIPTION
**Pull request by seancribbs**
This PR does two things:
1. Deprecates Fedora 15 for versions past 1.2. 1.3+ will use Fedora 17.
2. Adds `previous` and `legacy` version tags for the "upgrade" tests, letting us parameterize whether we're upgrading from `current - 1` or `current - 2`.

If this is not merged right away, I will likely adjust the bubbles in the UI to reflect the new tag and add a commit for that.

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -11,7 +11,7 @@
 #
 # It's strongly recommended to check this file into your version control system.
```
Comment (joedevivo) [inline db/schema.rb:12]: Well done!

Comment (seancribbs) [inline db/schema.rb:12]: :fish:

File: assets/javascripts/app/views.js [hunk 2]
```diff
@@ -52,24 +52,36 @@ GiddyUp.ScorecardSubcellView = Ember.View.extend({
...
+      abbr = 'M'+abbr;
       break;
     default:
-      return 'U';
+      break;
     }
-
-    return true;
-  }.property('content.test.backend')
+    if(abbr.length == 0)
```
Comment (cmeiklejohn) [inline assets/javascripts/app/views.js]: Three equals?

Comment (joedevivo) [inline assets/javascripts/app/views.js]: Three wolves?

http://www.amazon.com/The-Mountain-Three-Short-Sleeve/dp/B002HJ377A

**Comment by joedevivo** [comment]
Looks good, but I didn't test anything. I can if you want, but we are to AGILE for that

**Comment by cmeiklejohn** [comment]
:+1:

**Comment by joedevivo** [comment]
Oh snap~

```
➜  giddyup git:(sdc-upgrade-tags) foreman run rake db:seed   
rake aborted!
undefined method `each' for "previous legacy":String
```

**Comment by cmeiklejohn** [comment]
bundle exec rake?

**Comment by joedevivo** [comment]
+1.5, keep the change

**Comment by cmeiklejohn** [comment]
keep the change, ya filthy animal.

**Comment by cmeiklejohn** [comment]
http://www.youtube.com/watch?v=4dGOfFbzvq4

**Pull request merged**